server/src/main/java/org/spine3/server/entity/RecordBasedRepository.java

```diff
@@ -284,9 +287,12 @@ protected E findOrCreate(I id) {
     /**
      * Converts the passed entity into the record.
      */
-    protected EntityRecord toRecord(E entity) {
+    protected EntityRecordWithColumns toRecord(E entity) {
         final EntityRecord entityRecord = entityConverter().convert(entity);
```

Comment:
The point is that the Guava `Converter` has all its `public` methods `@Nullable`.
I'll add `checkNotNull`.